Libraries/CameraRoll/RCTImagePickerManager.m

```diff
@@ -17,6 +17,8 @@
 #import <React/RCTRootView.h>
 #import <React/RCTUtils.h>
 
+@import Photos;
```

Comment:
dont think we need this since we already `#import <Photos/Photos.h>` earlier?

Reply:
`PHFetchResult` and `PHAsset` depends on this and if we don't have it the compiler can't find the definitions :\ idk tbh but a quick google said we had to do this